core/src/components/card/test/shape/card.e2e.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * Shape is only available in the Ionic theme.
+ */
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('card: shape'), () => {
+    test.describe('default', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.goto(`/src/components/card/test/shape`, config);
+
+        await page.setIonViewport();
```

Comment:
```suggestion
```

Since we are not taking a screenshot of the entire page we do not need this call. Same goes for all other instances of this where we do not take a screenshot of the entire page.

Reply:
Removed: https://github.com/ionic-team/ionic-framework/pull/29292/commits/3c81ea03f67b5aa092a96b136829e764246adbd0